config.json

```diff
@@ -1386,418 +1400,418 @@
   },
   "concepts": [
     {
-        "uuid": "f7be1969-380e-4348-ac79-9b2834a526fd",
```

Comment:
nit: I guess we had a different level of indentation? Not a blocker but I would prefer we not have extraneous changes like this.